core/script.runtime/src/kotlin/script/dependencies/resolvers.kt

```diff
@@ -21,6 +21,24 @@ package kotlin.script.dependencies
 import java.io.File
 import kotlin.script.dependencies.DependenciesResolver.ResolveResult
 
+//discuss
+
+// Provides api to discover dependencies of scripts
+// Dependencies can be content-dependent
+//
+// Some concerns on naming:
+// Environment -> ScriptEnvironment (top level class with too common a name)
+// ResolveResult -> ResolutionResult
```

Comment:
Noun+noun is more correct grammatically and is wider used in our project (for example, `AnalysisResult`, `CompilationResult`, `DeadCodeEliminationResult`, `FileProcessingResult`, etc.)